data/hooks/conf_regen/25-dovecot

```diff
@@ -27,6 +27,13 @@ do_pre_regen() {
       "${dovecot_dir}/dovecot.conf"
   fi
 
+  # Allow to configure pop3 protocol support
+  export pop3_enabled="$(yunohost settings get 'pop3.enabled')"
+  if [ ! -z "${pop3_enabled}" ]; then
+    ynh_package_install dovecot-pop3d
```

Comment:
Uuuuuh that line feels weird ... Don't we want to do this instead in a setting hook ? c.f. the end of settings.py. Which you might also want to use to trigger a regenconf for dovecot when the user changes the settings